webdriver-factory/src/main/scala/io/magentys/cinnamon/webdriver/factory/RemoteWebDriverFactory.scala

```diff
@@ -0,0 +1,25 @@
+package io.magentys.cinnamon.webdriver.factory
```

Comment:
Remove this RemoteWebDriverFactory singleton and move the logic back into the WebDriverFactory. Switch to use pattern matching, i.e.

```
if (hubUrl.isDefined && !hubUrl.get.isEmpty) {
      capabilities.getCapability("platformName") match {
        case "Android" => new AndroidDriver(new URL(hubUrl.get), capabilities)
        case "iOS" => new IOSDriver(new URL(hubUrl.get), capabilities)
        case _ => new RemoteWebDriver(new URL(hubUrl.get), capabilities)
      }
    }
```